social_dilemmas/envs/pettingzoo_env.py

```diff
@@ -0,0 +1,72 @@
+from gym.utils import EzPickle
+from pettingzoo.utils import wrappers
+from pettingzoo.utils.conversions import from_parallel_wrapper
+from pettingzoo.utils.env import ParallelEnv
+
+from social_dilemmas.envs.env_creator import get_env_creator
+
+MAX_CYCLES = 1000
```

Comment:
Is this number standard in SSD? Where did it come from?

Reply:
Yes, it's the 'horizon' from the RLlib implementation in run_scripts/train.py-RLlib resets environments at the end of each rollout of length 1000, so I've put in the same for SSD.